cloud/disk_manager/internal/pkg/dataplane/url/vhd/footer.go

```diff
@@ -5,39 +5,57 @@ import "github.com/ydb-platform/nbs/cloud/disk_manager/internal/pkg/dataplane/ur
 ////////////////////////////////////////////////////////////////////////////////
 
 const (
-	footerCookie = "conectix"
+	footerCookie        = "conectix"
+	fileFormatVersion   = uint32(0x00010000)
+	dynamicHardDiskType = uint32(3)
 )
 
 ////////////////////////////////////////////////////////////////////////////////
 
 // https://learn.microsoft.com/en-us/windows/win32/vstor/about-vhd
 type footer struct {
 	Cookie             [8]byte
-	Features           uint32
-	FileFormatVersion  uint32
-	DataOffset         uint64
-	Timestamp          uint32
-	CreatorApplication uint32
-	CreatorVersion     uint32
-	CreatorHostOS      uint32
-	OriginalSize       uint64 // Stores the size of the hard disk in bytes at creation time.
-	CurrentSize        uint64 // Stores the current size of the hard disk in bytes.
-	DiskGeometry       uint32
-	DiskType           uint32
-	Checksum           uint32
-	UniqueID           [16]byte
-	SavedState         byte
+	Features           uint32   // A bit field used to indicate specific feature support.
+	FileFormatVersion  uint32   // Is divided into a major/minor version and matches the version of the specification used in creating the file.
+	DataOffset         uint64   // Absolute byte offset, from the beginning of the file, to the next structure.
+	Timestamp          uint32   // Creation time of a hard disk image.
+	CreatorApplication [4]byte  // Used to document which application created the hard disk.
```

Comment:
the hard disk image?

Reply:
this comments are from official microsoft documentation

so, they use "hard disk" term. btw VHD is virtual HARD disk format